### ISSUE TITLE
Linux binary distribution has Gnome incompatibilities with Wayland under modern distros.

### ISSUE THREAD

**Issue by ggarra13** [ggarra13/mrv2#240]
**Describe the bug**
The binary Linux distribution of mrv2 may run into some Wayland incompatibilities with modern distros like Ubuntu 22.04.4 LTS when run under Gnome:

**To Reproduce**
1. Install the binary distribution compiled on Rocky Linux 8.9.
2. Login into Wayland
3. Start up mrv2 from the command-line.
4. See error:

```
(mrv2:540344): GLib-GIO-ERROR **: 20:47:45.978: Settings schema 'org.gnome.settings-daemon.plugins.xsettings' does not contain a key named 'antialiasing'
Trace/breakpoint trap (core dumped)
```

**Expected behavior**
Not crash.

To fix it, you may need to do a hack, like:
```
sudo cp /usr/share/glib-2.0/schemas/org.gnome.settings-daemon.plugins.xsettings.gschema.xml /usr/share/glib-2.0/schemas/org.gnome.settings-daemon.plugins.xsettings.gschema.xml.bad

sudo nano /usr/share/glib-2.0/schemas/org.gnome.settings-daemon.plugins.xsettings.gschema.xml
    (remove lines 19 and 20)
        <   </schema>
        <   <schema id="org.gnome.settings-daemon.plugins.xsettings.deprecated">

sudo glib-compile-schemas /usr/share/glib-2.0/schemas
```

**Desktop (please complete the following information):**
 - OS: Linux Wayland
 - Version: mrv2 v1..17

**Comment by ggarra13** [comment]
v1.1.8 .rpm and .deb installers will try to fix the deprecated functionality as part of the install.

**Comment by ggarra13** [comment]
Fix in v1.1.8.
